website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx

```diff
@@ -12,7 +12,7 @@ Policies are rules that Terraform Cloud enforces on Terraform runs. You can defi
 @include 'tfc-package-callouts/policies.mdx'
 <!-- END: TFC:only name:pnp-callout -->
 
-Policy sets are collections of policies you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces in your organization. For each run in the selected workspaces, Terraform Cloud checks the Terraform plan against the policy set. Depending on the [enforcement level](#policy-enforcement-levels), failed policies can stop a run in a workspace.
+Policy sets are collections of policies you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces in your organization. For each run in the applicable workspaces, Terraform Cloud checks the Terraform plan against the policy set. Depending on the [enforcement level](#policy-enforcement-levels), failed policies can stop a run in a workspace. You can exclude selected workspaces from a policy set if you don't want Terraform Cloud to check those workspaces' runs against the policy set.
```

Comment:
I'm suggesting this shortened version because we do expand on this below, and this keeps the paragraph a bit slimmer:
```suggestion
Policy sets are collections of policies you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces in your organization. For each run in the applicable workspaces, Terraform Cloud checks the Terraform plan against the policy set. Depending on the [enforcement level](#policy-enforcement-levels), failed policies can stop a run in a workspace. If you do not want to enforce a policy set on a specific workspace, you can exclude the workspace from that set.
```

Reply:
👋🏽 Jeff is now on PTO so I'm taking over this PR - that sounds good, Rose!